File: ruby/merge_sort/test_merge_sort.rb
require_relative 'merge_sort'
require 'test/unit'

class TestMergeSort < Test::Unit::TestCase
  def test_sort
    test_list = [21, 4, 1, 3, 9, 20, 25]
    assert_equal(test_list.sort, sort(test_list))
  end
end

File: python/merge_sort/test_merge_sort.py
import unittest
from merge_sort import sort

class MergeSortTest(unittest.TestCase):
    def test_sort(self):
        test_list = [21, 4, 1, 3, 9, 20, 25]
        self.assertItemsEqual(sort(test_list), [1, 3, 4, 9, 20, 21, 25])

if __name__ == '__main__':
    unittest.main()
